nextflow.config

```diff
@@ -67,11 +67,12 @@ profiles {
         cleanup                = false
         nextflow.enable.configProcessNamesValidation = true
     }
-    wave {
-        wave.enabled = true
-        wave.strategy = ['conda', 'container', 'dockerfile', 'spack']
-        wave.build.repository = 'quay.io/seqeralabs/nf-aggregate'
-        wave.freeze = true   
+    seqera_containers {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
```

Comment:
```suggestion
```

Not needed. Those forces the use of Docker image what's converted by Singularity